Makefile

```diff
@@ -20,10 +20,10 @@ ifneq ($(IN_DOCKER),)
 	docker_cmd := env http_proxy= https_proxy= docker
 	compose_cmd := env http_proxy= https_proxy= docker-compose
 	common_run := $(compose_cmd) run --rm $(run_args)
-	run := $(common_run) -p 9636:9636 shell
+	run := $(common_run) shell
 	docs_host := ${DOCKER_HOST}
 	docs_run := $(common_run) -p 9633:9633 shell
-	web_dev_run := $(common_run) -p 9636:9636 shell
```

Comment:
That's why it was added. If you do `make start-web-dev`, you can hit the API and web app on 9636.

Before this I was running with the ugly hack Timberman showed me:

`env HAB_DEPOT_URL="https://willem.habitat.sh/v1/depot -p 9636:9636" make shell`

(notice how the quotes are set up on that var, to, with trickery, pass `-p` into `docker-compose`.)

What I've been doing since running multiple `make shell`s was broken is doing `docker run -it ID bash` to open new shells in the same container.

I don't have much of an opinion on how to do things here, just showing how I managed to get by before and after.